### ISSUE TITLE
Function qa_format_number() adds decimal, even if $decimals = 0

### ISSUE THREAD

**Issue by q2apro** [q2a/question2answer#781]
It is more a logical issue, not a bug per se.

In `function qa_format_number($number, $decimals = 0, $compact = false)` https://github.com/q2a/question2answer/blob/dev/qa-include/app/format.php#L2366 there are the lines:

		// keep decimal part if not 0 and number is short (e.g. 9.1k)
		$rounded = round($number, 1);
		if ($number < 100 && ($rounded != (int)$rounded)) {
			$decimals = 1;
		}

*Even though decimals is passed with 0.*

This leads, for instance, to inconsistency in the users page:

![image](https://user-images.githubusercontent.com/6764548/67619327-c594d800-f802-11e9-8ce6-125e7b243927.png)

In my opinion, it should be optional or avoided. – Or do this for numbers under 10k and add a `,0` too, to have it consistent.

I suggest (**but only for the users page**): 

```
		if ($number < 10) {
			$decimals = 1;
		}
```

If this is set global, e. g. the answer count displays as "0.0" 🌝

**Comment by svivian** [comment]
The `$decimals` is for showing actual decimal numbers, but "compact numbers" are actually whole numbers behind, just shortened. So that parameter has no effect in the case of compact numbers.

I can see value in having the `.0` there for consistency. To me it makes sense to keep the decimal for 10k-99k so we have 3 significant figures. I also wonder if we need to shorten numbers below 10k at all. Having 4 digits (plus comma/period) doesn't take up too much space.

So in other words we'd have
155k
136k
91.6k
10.0k
9,876
4,321

What do you think?

**Comment by q2apro** [comment]
I would go for:

155k
136k
92k
10k
9.8k
4.3k
2.0k
1.2k
1.0k

and below either 0.1k to 0.9k or 1 to 999.
